Tests/Singulink.Globalization.Currency.Tests/SortedMoneySetTests/TryGetValueTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Shouldly;
+
+namespace Singulink.Globalization.Tests.SortedMoneySetTests;
+
+[TestClass]
+public class TryGetValueTests
+{
+    private static readonly Money _usd100 = new Money(100m, "USD");
+    private static readonly Money _eur50 = new Money(50m, "EUR");
+    private static readonly Money _cad25 = new Money(25m, "CAD");
+    private static readonly ImmutableSortedMoneySet _moneySetValues = new(_usd100, _eur50, _cad25);

```

Comment:
Follow the same pattern as the other tests - static `_immutableSet` and instance `_set` fields